code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -27,32 +29,39 @@
 	source = null
 	movetotarget = 1
 
+
 /obj/effect/accelerated_particle/weak
 	movement_range = 8
-	energy = 5
+	energy = PARTICLE_ENERGY / 2
```

Comment:
Defining it as anything *other* than PARTICLE_ENERGY / 2 would defeat the whole purpose of defines anyway
And why define a number used in only one place ever?